packages/craco/README.md

```diff
@@ -673,10 +677,13 @@ A few utility functions are provided by CRACO to help you develop a plugin:
  - `addBeforeLoaders`
  - `addAfterLoader`
  - `addAfterLoaders`
+ - `getPlugin`
+ - `removePlugins`
+ - `addPlugins`
```

Comment:
A replacePlugin function is missing.

Reply:
The remove and add functions should be enough. A replace function would make it confusing in my opinion